shared/constants/wallets.js

```diff
@@ -449,30 +449,32 @@ export const updatePaymentsReceived = (
   )
 }
 
-export const acceptDisclaimerWaitingKey = 'wallets:acceptDisclaimer'
-export const changeAccountNameWaitingKey = 'wallets:changeAccountName'
-export const createNewAccountWaitingKey = 'wallets:createNewAccount'
-export const changeDisplayCurrencyWaitingKey = 'wallets:changeDisplayCurrency'
-export const getDisplayCurrencyWaitingKey = (id: Types.AccountID) => `wallets:getDisplayCurrency:${id}`
-export const linkExistingWaitingKey = 'wallets:linkExisting'
-export const loadEverythingWaitingKey = 'wallets:loadEverything'
-export const buildPaymentWaitingKey = 'wallets:buildPayment'
-export const sendPaymentWaitingKey = 'wallets:stellarSend'
-export const requestPaymentWaitingKey = 'wallets:requestPayment'
-export const setAccountAsDefaultWaitingKey = 'wallets:setAccountAsDefault'
-export const deleteAccountWaitingKey = 'wallets:deleteAccount'
+const p = 'wallets:'
+export const acceptDisclaimerWaitingKey = `${p}acceptDisclaimer`
```

Comment:
i believe this ruins the flow typing

Reply:
How so? Flow still succeeds...

Reply:
in the past flow got confused by this vs static strings.maybe its fixed, maybe not. it also depends on how it looks from outside the file. aka is this exported as a string or the constant etc

Reply:
Okay, changed it back.